src/roi/Roi.ts

```diff
@@ -626,3 +654,103 @@ function getBorders(roi: Roi): Border[] {
     };
   });
 }
+
+function getEllipse(roi: Roi, scale: number): Ellipse {
```

Comment:
Please move this function to another file (in a subfolder: `./properties/getEllipse`. Roi.ts is becoming really too big.